edi_oca/models/edi_exchange_record.py

```diff
@@ -383,7 +414,15 @@ def action_open_related_record(self):
         return self.record.get_formview_action()
 
     def _set_related_record(self, odoo_record):
-        self.sudo().update({"model": odoo_record._name, "res_id": odoo_record.id})
+        self._set_related_records(odoo_record)
```

Comment:
I would deprecate `_set_related_record` in favor of `_set_related_records`.

Reply:
how would you handle it? should we adapt the already existing '_set_related_record' to handle multiple odoo_record?